reflex/.templates/apps/blank/code/blank.py

```diff
@@ -11,24 +11,31 @@
 class State(rx.State):
     """The app state."""
 
+    ...
+
 
 def index() -> rx.Component:
-    return rx.center(
-        rx.theme_panel(),
+    # Welcome Page (Index)
+    return rx.container(
+        rx.color_mode.button(position="top-right"),
         rx.vstack(
             rx.heading("Welcome to Reflex!", size="9"),
-            rx.text("Get started by editing ", rx.code(filename)),
-            rx.button(
+            rx.text("Get started by editing ", rx.code(filename), size="5"),
+            rx.link(
```

Comment:
Can we put the button inside the link, so we don't need the style props?

Reply:
oh yeah, that's really nice!

```python
            rx.link(
                rx.button("Check out our docs!"),
                href=docs_url,
                is_external=True,
            ),
```